Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -46,66 +157,35 @@ public struct ConsoleLogger: LogHandler {
         set { self.metadata[key] = newValue }
     }
     
-    /// See `LogHandler.log(level:message:metadata:file:function:line:)`.
+    /// See `LogHandler.log(level:message:metadata:source:file:function:line:)`.
```

Comment:
To avoid a public API break, the original version needs to be provided and call through to the new one.
```suggestion
    /// See `LogHandler.log(level:message:metadata:file:function:line:)`.
    public func log(
        level: Logger.Level,
        message: Logger.Message,
        metadata: Logger.Metadata?,
        file: String,
        function: String,
        line: UInt
    ) {
        self.log(level: level, message: message, metadata: metadata, source: file.prefix(while: { $0 != "/"  }), file: file, function: function, line: line)
    }
    
    /// See `LogHandler.log(level:message:metadata:source:file:function:line:)`.
```

Reply:
The `LogHandler` protocol provides this already, so I don't think this is actually an API break? Unless there's some obscure way to reference the method that can't see the protocol extension method.

https://github.com/apple/swift-log/blob/main/Sources/Logging/LogHandler.swift#L202

Reply:
That'd normally be sufficient, but the API breakage checker (see https://github.com/vapor/console-kit/actions/runs/5909305623/job/16030244244?pr=182) is too stupid to realize that, and it's easier to just provide the redundant override than it is to make the checker shut up. That being said, I'm in agreement that it's weird and annoying - I'd be fine with putting `#if CI` around the additional method to satisfy the checker while calling through to the overload in swift-log in other environments.